### ISSUE TITLE
2019 1-yr PUMS returning repweights as char

### ISSUE THREAD

**Issue by jdsher** [walkerke/tidycensus#325]
Easy to fix on the user end, but just thought it warranted a heads-up, the call:

```
get_acs(variables = "TEN",
              state = "WA",
              year = 2019,
              survey = "acs1",
              rep_weights = "housing")
```
returns a dataframe with the rep weights as character instead of numeric, thus causing ```to_survey()``` to fail.

**Comment by mfherman** [comment]
Thanks for the report, @jdsher! Interesting that it's doing this with 2019 data but not previous years. Will make the change to convert these columns returned by the API to doubles.

**Comment by walkerke** [comment]
@jdsher I'm seeing the replicate weights coming through as numeric.  Would you mind pasting the results of your `sessionInfo()`?

**Comment by jdsher** [comment]
Sure:

`R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] srvyr_0.4.0       survey_4.0        survival_3.2-7    Matrix_1.2-18     janitor_2.0.1    
 [6] scales_1.1.1      santoku_0.5.0     tigris_1.0        mapview_2.9.4     tidycensus_0.10.2
[11] sf_0.9-6          forcats_0.5.0     stringr_1.4.0     dplyr_1.0.2       purrr_0.3.4      
[16] readr_1.4.0       tidyr_1.1.2       tibble_3.0.3      ggplot2_3.3.2     tidyverse_1.3.0  

loaded via a namespace (and not attached):
 [1] httr_1.4.2         splines_4.0.3      jsonlite_1.7.1     modelr_0.1.8      
 [5] assertthat_0.2.1   sp_1.4-2           stats4_4.0.3       blob_1.2.1        
 [9] cellranger_1.1.0   pillar_1.4.6       backports_1.1.10   lattice_0.20-41   
[13] glue_1.4.2         uuid_0.1-4         digest_0.6.25      snakecase_0.11.0  
[17] rvest_0.3.6        colorspace_1.4-1   htmltools_0.5.0    pkgconfig_2.0.3   
[21] broom_0.7.1        raster_3.3-13      haven_2.3.1        webshot_0.5.2     
[25] satellite_1.0.2    generics_0.0.2     ellipsis_0.3.1     withr_2.3.0       
[29] cli_2.0.2          magrittr_1.5       crayon_1.3.4       readxl_1.3.1      
[33] maptools_1.0-2     fs_1.5.0           fansi_0.4.1        xml2_1.3.2        
[37] foreign_0.8-80     class_7.3-17       tools_4.0.3        mitools_2.4       
[41] hms_0.5.3          lifecycle_0.2.0    munsell_0.5.0      reprex_0.3.0      
[45] compiler_4.0.3     e1071_1.7-3        rlang_0.4.7        classInt_0.4-3    
[49] units_0.6-7        rstudioapi_0.11    rappdirs_0.3.1     htmlwidgets_1.5.2 
[53] crosstalk_1.1.0.1  leafem_0.1.3       base64enc_0.1-3    gtable_0.3.0      
[57] codetools_0.2-16   curl_4.3           DBI_1.1.0          R6_2.4.1          
[61] lubridate_1.7.9    rgdal_1.5-16       KernSmooth_2.23-17 stringi_1.5.3     
[65] Rcpp_1.0.5         vctrs_0.3.4        png_0.1-7          leaflet_2.0.3     
[69] dbplyr_1.4.4       tidyselect_1.1.0`

**Comment by walkerke** [comment]
@jdsher I'm seeing the same issue as you when using tidycensus 0.10.2, but it is corrected as of the current GitHub version (0.10.3).  Try installing from GitHub and see if that fixes it for you; we'll look to get a new version up on CRAN soon.
